packages/edit-navigation/src/components/notices/index.js

```diff
@@ -16,17 +11,15 @@ export default function EditNavigationNotices() {
 		( select ) => select( noticesStore ).getNotices(),
 		[]
 	);
-	const dismissibleNotices = filter( notices, {
-		isDismissible: true,
-		type: 'default',
-	} );
-	const nonDismissibleNotices = filter( notices, {
-		isDismissible: false,
-		type: 'default',
-	} );
-	const snackbarNotices = filter( notices, {
-		type: 'snackbar',
-	} );
+	const dismissibleNotices = notices.filter(
```

Comment:
Notices will always be an array:

https://github.com/WordPress/gutenberg/blob/af7da80dd54d7fe52772890e2cc1b65073db9655/packages/notices/src/store/selectors.js#L73

This is similar to how we handled notices in #46059.

Reply:
This package will likely be removed so we're all good here.